src/pages/komodo-defi-framework/api/v20/withdraw/index.mdx

```diff
@@ -561,6 +562,37 @@ This method generates a raw transaction which should then be broadcast using [se
   You can see the memo is included on the [block explorer](https://irishub.iobscan.io/#/txs/E00982A2A8442D7140916A34E29E287A0B1CBB4B38940372D1966BA7ACDE5BD6)
 </CollapsibleSection>
 
+##### Withdraw Tendermint IBC coins
+
+<CodeGroup title="Withdraw" tag="POST" label="withdraw" mm2MethodDecorate="true">
+  ```json
+  {
+    "mmrpc": "2.0",
+    "userpass": "RPC_UserP@SSW0RD",
+    "method": "withdraw",
+    "params": {
+      "coin": "ATOM-IBC_NUCLEUSTEST",
+      "to": "nuc16ggf7pmy0w0glzyrgd2x9mlw92csw6d9yj2qvg",
```

Comment:
This should make it more clear

```suggestion
      "coin": "ATOM",
      "to": "iaa1e0rx87mdj79zejewuc4jg7ql9ud2286g2us8f2",
```

Reply:
its making me more confused :/
The example is for an IBC withdraw, so shouldnt  it be an IBC token?
The coin is ATOM, but the address is in IRIS format?